src/components/ec-text-filter/ec-text-filter.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div
+    class="ec-text-filter"
+    :data-test="$attrs['data-test'] ? `${$attrs['data-test']} ec-text-filter` : 'ec-text-filter'"
+  >
+    <ec-input-field
+      v-model="inputModel"
+      v-bind="inputProps"
+      :left-icon="IconName.SimpleSearch"
+      :type="InputFieldType.TEXT"
+      :icon="rightIcon"
+      @icon-click="emit(TextFilterEvent.CHANGE, '')"
```

Comment:
I am not sure if this is a change event. Usually we have an an update function and an `update:modelValue` event. i.e https://github.com/Ebury/chameleon/blob/master/src/components/ec-currency-filter/ec-currency-filter.vue#L62

Reply:
I add update:modelValue as well